pkg/materialize/cluster.go

```diff
@@ -28,6 +30,27 @@ func NewClusterBuilder(conn *sqlx.DB, obj MaterializeObject) *ClusterBuilder {
 	}
 }
 
+type SchedulingConfig struct {
+	onRefresh               bool
+	rehydrationTimeEstimate string
+}
+
+func GetSchedulingConfig(v interface{}) SchedulingConfig {
+	if v == nil || len(v.([]interface{})) == 0 {
+		return SchedulingConfig{}
+	}
```

Comment:
Why not type `v` as `[]interface{}`?  I noticed that another function calls GetSchedulingConfig with a param that's of type `v interface{}` but wouldn't that code path panic here?